web/src/auth/LoginPage.js

```diff
@@ -379,6 +380,12 @@ class LoginPage extends React.Component {
       return;
     }
     if (this.state.loginMethod === "password" || this.state.loginMethod === "ldap") {
```

Comment:
Will `this.state.loginMethod === "ldap"` condition work for you?

Reply:
yes, The backend's LDAP login processing includes the password parameter (controllers\auth.go:474~481)